src/main/kotlin/com/mparticle/kits/MParticleAutopilot.kt

```diff
@@ -102,7 +100,17 @@ class MParticleAutopilot : Autopilot() {
             val accentColor = configuration.notificationColor
             if (!UAStringUtil.isEmpty(accentColor)) {
                 try {
-                    editor.putInt(NOTIFICATION_COLOR, Color.parseColor(accentColor))
+                    val colorValue = when {
+                        accentColor.equals("System default") -> {
```

Comment:
Are you sure that "System Default" is the only potential string option here? I see that we are choosing a default color in the case of this specific value, but in other cases of parse failure (such as a hypothetical "User Default" or something else) we use no color at all? Does that cause visibility problems? Should we default to the system default also any time `Color.parseColor()` fails?

Reply:
By default, the current code uses a transparent color. I'll also add a fallback option.

Reply:
this is  client's comment `we default to NotificationCompat.COLOR_DEFAULT, which is Color.TRANSPARENT . the NotificationCompat docs say that means that the system won't decorate the notification with a custom color and will use the system default colors when presenting the notification`. so, mParticle is currently using a default transparent color, so we don't need to handle this anymore.